playbooks/kvm/create.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Process KVM Nodes
+  import_playbook: process.yml
+  when: (setup_vms | default(true))
+
+- name: Provision VMS
+  hosts: vm_hosts
+  vars:
+    SETUP_VMS: "{{ setup_vms | default((kvm_nodes | default([])) | length | int >= 1) }}"
+  tasks:
+    - name: Validate vm_host
+      import_role:
+        name: kvm
+        tasks_from: validate
+    - name: Destroy VMs
+      import_role:
+        name: kvm
+        tasks_from: destroy
+      when: SETUP_VMS | bool
+      tags:
+        - destroy_vms
+    - name: Setup VM Host Network
+      import_role:
+        name: kvm
+        tasks_from: setup_up_host_network.yml
```

Comment:
This is the only `tasks_from` entry with the `.yml` in this file. Many other files include the `.yml`. Please make this standard across all uses.

I slightly prefer having the `.yml` as it makes it clear that it's loading a specific file, not doing any black magic under the hood akin to tags.